newIDE/app/src/SceneEditor/index.js

```diff
@@ -903,6 +903,7 @@ export default class SceneEditor extends React.Component<Props, State> {
       properties: {
         type: 'secondary',
         title: <Trans>Properties</Trans>,
+        tooltip: t`Properties of Instance`,
```

Comment:
Let's go for `Display the properties of the selected instances from the scene editor`.